### PR TITLE
Features/add esys docs

### LINKED ISSUE #113: Document energy system model
Add to [esys docs](https://github.com/rl-institut-private/digipipe/blob/489447f93a4d10c432cfc1c0281e5039697042c2/docs/sections/esys.md):
- esys topology (image)
- table with esys components (the one from the Wolke, maybe reduced)
- refer to BMWK long term scenarios
- simplifications
  - time resolution
  - perfect foresight
  - aggregation (ABW, central+decentral, ...)

### PR DESCRIPTION
**Pull request by MaGering**
Fixes #113

## Before merging into `dev`-branch, please make sure that the following points are checked:

- [x] All pre-commit tests passed locally with: `pre-commit run -a`
- [x] File `CHANGELOG.md` was updated
- [x] The docs were updated
  - ~[ ] if `dataset.md`s were updated, the dataset docs were regenerated using
    `python docs/generate_dataset_mds.py`~

If packages were modified:
- ~[ ] File `poetry.lock` was updated with: `poetry lock`~
- ~[ ] A new env was successfully set up~

WARNING: When modifying use snakemake <=7.32.0, cf. #186

If data flow was adjusted:
- ~[ ] Data pipeline run finished successfully with: `snakemake -jX`~
- ~[ ] Esys appdata was created successfully with: `snakemake -jX make_esys_appdata`~

  (with `X` =  desired number of cores, e.g. 1)

### CODE REVIEW

File: docs/sections/esys.md [hunk 1]
```diff
@@ -1,7 +1,196 @@
...
+- Für das Jahr 2045 ist kein Import von Braunkohle vorhergesehen.
+- Es wird das Jahr 2045 in stündlichen Schritten betrachtet.
+- Sämtliche Zeitreihen liegen in dieser Auslösung vor oder werden in diese
+  Auflösung übersetzt.
+- Es handelt sich um eine Dispatch Optimierung.
+- Sämtliche Kapazitäten bis auf die der Importe und Exporte sowie die der
+  Speichertechnologien sind fix gesetzt.
+- Die Optimierung findet nach dem Perfect forsight Prinzip über den
+  Zeithorizont eines Jahres statt.
+- TODO: aggregation (ABW, central+decentral, ...)
```
Comment (MaGering) [inline docs/sections/esys.md:49]: I was not sure about this. Partly it is already explained in the md files of the data processing itself. What else would you like to have mentioned on this point @nesnoj?

**Comment by MaGering** [comment]
First draft of esys documentation is ready for review. Let me know what you think @nesnoj

File: docs/sections/esys.md [hunk 2]
```diff
@@ -1,7 +1,196 @@
...
+
+## Annahmen und Vereinfachungen
+
+Wir treffen die folgenden Annahmen und Vereinfachungen:
+
+- Für die Prognosen werden [Langfristszenarien](https://langfristszenarien.de/enertile-explorer-de/index.php)
+  verwendet.
+- Biomasse wird unterteilt in: feste, gasförmige und flüssige Biomasse.
+- Zur Abbildung der Biomasse wird vom Status-quo in ABW ausgegangen.
+- Der Status-qup ist das Jahr 2022.
```
Comment (MaGering) [inline docs/sections/esys.md:26]:
```suggestion
- Der Status-quo ist das Jahr 2022.
```

File: docs/sections/esys.md [hunk 3]
```diff
@@ -1,7 +1,196 @@
...
+
+## Annahmen und Vereinfachungen
+
+Wir treffen die folgenden Annahmen und Vereinfachungen:
+
+- Für die Prognosen werden [Langfristszenarien](https://langfristszenarien.de/enertile-explorer-de/index.php)
+  verwendet.
+- Biomasse wird unterteilt in: feste, gasförmige und flüssige Biomasse.
+- Zur Abbildung der Biomasse wird vom Status-quo in ABW ausgegangen.
+- Der Status-quo ist das Jahr 2022.
```
Comment (MaGering) [inline docs/sections/esys.md:26]: Is this correct @nesnoj?

File: docs/sections/esys.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (nesnoj) [reply docs/sections/esys.md]: Almost, I can fix it

**Comment by nesnoj** [comment]
> First draft of esys documentation is ready for review. Let me know what you think @nesnoj

Thanks a lot for this docs part! :pray: 
I'll check ...

**Review by nesnoj — APPROVED** [review]

**Pull request merged**